sdk/search/Azure.Search.Documents/tests/DocumentOperations/AutocompleteTests.cs

```diff
@@ -28,7 +28,6 @@ public AutocompleteTests(bool async, SearchClientOptions.ServiceVersion serviceV
             Assert.NotNull(expectedText);
             Assert.NotNull(expectedQueryPlusText);
 
-            // TODO: #16824 - investigate autocompletions across SKUs
             CollectionAssert.IsSubsetOf(completions.Results.Select(c => c.Text), expectedText);
             CollectionAssert.IsSubsetOf(completions.Results.Select(c => c.QueryPlusText), expectedQueryPlusText);
```

Comment:
The important change here was from `CollectionAssert.AreEqual` to `IsSubsetOf` - have we investigated why that changed between SKUs?